src/main/java/frc/robot/RobotContainer.java

```diff
@@ -69,8 +76,16 @@ public RobotContainer() {
    * joysticks}.
    */
   private void configureBindings() {
-    driverXbox.a().onTrue((Commands.runOnce(drivebase::zeroGyro)));
-    driverXbox.b().onTrue(Commands.none());
+
+    driverXbox.a().whileTrue(pivotSubsystem.PivotDown(pivotSubsystem, 0.75, Degrees.of(-6)));
```

Comment:
```suggestion
    driverXbox.a().whileTrue(pivotSubsystem.PivotDown(pivotSubsystem, 0.75, Degrees.of(-6)));
```

You should put 0.75 in a constant and use a lower value for initial testing.